docs/reference/modules/admin-bar.md

```diff
@@ -25,6 +25,8 @@ The options passed in this manner will configure the existing @apostrophe/admin-
 The `groups` option takes an array of one or more objects that group several menu items together in the admin bar as a dropdown menu. Each of the `groups` objects requires a `label` and an array of menu `items`. The `label` will be used as the label displayed in the menu. The `items` array contains the names of the individual menu items you want to appear in the dropdown, entered in the order you want them to appear. Note: Menu names for `piece-type` items are the name of the piece-type, not the label. For core items, like 'Images', the name is prefixed - '@apostrophecms/image'.
 
 **Example**
+
+<AposCodeBlock>
 ```javascript
 // modules/@apostrophecms/admin-bar/index.js
```

Comment:
Check out other uses of this component. There's a special slot for the file name.

Reply:
Ahh - I understand your other comment now. Thanks.